Installing-Oppia-(Mac-OS;-Python-3).md

```diff
@@ -9,62 +9,81 @@
+2. Next, we will create a Rosetta terminal that emulate the *Intel* architecture. To do so, open a new terminal and run the following command (change "bash" to "zsh" if you're using a zsh terminal):
 
-    ```console
-    $ curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip2.py
-    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
-                                   Dload  Upload   Total   Spent    Left  Speed
-     100 1863k  100 1863k    0     0  16.7M      0 --:--:-- --:--:-- --:--:-- 18.1M
+    ```shell
+    $ /usr/bin/arch -x86_64 $SHELL --login
     ```
 
-    * Add the following to the `$PATH` variable on your Mac: `/Library/Frameworks/Python.framework/Versions/2.7/bin`
+    This switches the architecture from Mac M1's *ARM* architecture to the emulated *Intel* architecture for the current session. To verify this, run `arch` in the terminal and you should see `i386` being printed. You will need to switch the architecture to Intel for all Oppia development.
 
-    * Download `plistlib.py`
+    If you use Homebrew to install any Python development dependencies for pyenv (discussed below), you will need to install and use Homebrew in this terminal as well. Note that while you can have a Homebrew installation for Apple Silicon and another for Intel architectures installed simultaneously, pyenv is not smart enough to pick the Intel dependencies if both are present, so we recommend using an Intel installation of Homebrew exclusively.
 
-   ```console
-   $ curl https://raw.githubusercontent.com/python/cpython/2.7/Lib/plistlib.py -o 
-   /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plistlib.py
-   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
-                                 Dload  Upload   Total   Spent    Left  Speed
-   100 15185  100 15185    0     0  44274      0 --:--:-- --:--:-- --:--:-- 45193
-   ```
-    * Now try doing `python --version` on your command-line, and you should see version `2.7` printed.
+3. Follow the installation instructions below in your Rosetta terminal.
 
-2. Ensure that you have [Python 3.8](https://www.python.org/downloads/release/python-3812/) installed (Note: you can check this by running `python3 --version`). If Python 3.8 is not installed, download and run the latest Python 3.8 installer from https://www.python.org/downloads/mac-osx/.
+## Install prerequisites
+
+Oppia relies on a number of programs and third-party libraries. Many of these libraries are downloaded automatically for you when you first run the `start.py` script provided with Oppia. However, there are some things that you will need to do beforehand:
+
+1. Install [version 8+ of the Java Runtime Environment (JRE)](https://www.java.com/en/download/).
 
-3. Download [git](http://git-scm.com/download/mac), then run the package and follow instructions. This allows you to store the source in version control.
+2. Install [direnv](https://direnv.net/).
```

Comment:
I believe the cmd is `brew install direnv`. Can we put that instead of saying install direnv?

Reply:
There are more steps than just `brew install direnv`, for example changes to *rc files to hook direnv into the user's shell. I link to the direnv instructions because they do a good job of explaining what is required, will be kept up to date if newer versions of direnv need different *rc file changes, and cover other shells besides bash (e.g. zsh, fish)

Reply:
Sounds good!

probably add a comment (if you can) to the wiki.